scripts-setup/setup-env-internal

```diff
@@ -59,7 +59,7 @@ $(machine_list)
 
```

Comment:
@ichergui should we make it clear that tegrademo-mender isn't supported yet?

Reply:
@dwalkes yes, I guess 
My cleaning was not focus on the mender support, I removed the machines that need `uboot` and they are not supported in `JetPack 5.0.2`

Reply:
This is fine for now.  We can handle the mender cleanup later.